dessia_common/forms.py

```diff
@@ -378,80 +378,80 @@ def maldefined_method(self, arg0, arg1=1, arg2: int = 10, arg3=3):
     # def method_with_faulty_typing(arg0: Iterator[int]):
     #     return arg0
 
-    def to_markdown(self):
```

Comment:
At first, I was OK to remove all this, but then I realised that it was a cool way to show how titles, lists, links, etc behave on the platform.

Couldn't we keep it and concatenate the super method to add the table at the bottom of it ?

Reply:
I did as you asked